CHANGES.md

```diff
@@ -8,6 +8,7 @@ Release Notes.
 * Chore: adapt `create_source_release.sh` to make it runnable on Linux.
 
 #### Java Agent
+* Update quartz-scheduler plugin.
```

Comment:
This changelog is not accurate, please update.

Reply:
Need to describe the operationName changes and catch exceptions?

e.g. Update quartz-scheduler plugin operationName and catch job exceptions.

Reply:
Such as 

> The operation name of quartz-scheduler plugin in xx case, have been changed as the yyy format.

Reply:
Is this appropriate?


> The operation name of quartz-scheduler plugin in logic span analysis case, have been changed as the `quartz-scheduler/${className}` format.

Reply:
The operation name of quartz-scheduler plugin, has been changed as the quartz-scheduler/${className} format.